native/datasets_extension.cc

```diff
@@ -219,29 +220,36 @@ void register_datasets_extension(py::module& root) {
         .def(
             "__getitem__",
             [python_patient, pickle](py::object self_object,
-                                     uint32_t patient_id) {
+                                     uint64_t patient_id) {
```

Comment:
Just curious is there a reason for the memory bump?

Reply:
OMOP person_ids (aka what we are switching to) are uint64_t. I should probably introduce a typedef to make things here much more explicit.